public/app/features/alerting/unified/api/alertmanagerApi.ts

```diff
@@ -72,9 +73,13 @@ export const alertmanagerApi = alertingApi.injectEndpoints({
         // TODO Add support for active, silenced, inhibited, unprocessed filters
         const filterMatchers = filter?.matchers
           ?.filter((matcher) => matcher.name && matcher.value)
-          .map(
-            (matcher) => `${wrapWithQuotes(matcher.name)}${matcherToOperator(matcher)}${wrapWithQuotes(matcher.value)}`
-          );
+          .map((matcher) => {
+            return encodeMatcher({
```

Comment:
nit: we can use `matcherToMatcherField()` here to avoid some code duplication – I've appended that to the PR